lxc/list.go

```diff
@@ -561,7 +562,7 @@ func (c *cmdList) parseColumns(clustered bool) ([]column, bool, error) {
 
 			k := cc[0]
 			if colType == configColumnType {
-				if _, err := shared.ConfigKeyChecker(k); err != nil {
+				if _, err := shared.ConfigKeyChecker(k, instancetype.Any); err != nil {
```

Comment:
I wondered about this earlier, so we should only allow lxc list to display common fields for all instance types?